assets/javascripts/discourse/initializers/custom-wizard-redirect.js.es6

```diff
@@ -15,30 +19,26 @@ export default {
       window.location.href = wizardUrl;
     });
 
-    const ApplicationRoute = requirejs("discourse/routes/application").default;
+    withPluginApi("0.8.36", (api) => {
+      api.onAppEvent("page:changed", (data) => {
+        const currentUser = container.lookup("service:current-user");
```

Comment:
@merefield currentUser lookup should either be at the start of the initialiser, or using the api. Preferably using the api.

Reply:
Moved to api